pkg/clusterstatus/apiserver/apiserver.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2019 The KubeOne Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	kubeoneapi "github.com/kubermatic/kubeone/pkg/apis/kubeone"
+	"github.com/kubermatic/kubeone/pkg/ssh"
+)
+
+const (
+	defaultHTTPTimeout = 10 * time.Second
+	healthzEndpoint    = "https://%s:6443/healthz"
+)
+
+type Status struct {
+	Health bool `json:"health,omitempty"`
+}
```

Comment:
I'm thinking about getting rid of this struct and just returning bool from a function.

Reply:
I think it's OK to keep this as a struct, probably we going to add some fields there (some metrics/stats?).